stylish.js

```diff
@@ -41,7 +41,7 @@ export default {
   formatResults: function (results, filename, options) {
     const {messages} = results
     let output = []
-    let underlinedFilename
+    let underlinedFilename = ''
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing semicolon.